lark/grammars/lark.lark

```diff
@@ -1,25 +1,39 @@
 # Lark grammar of Lark's syntax
 # Note: Lark is not bootstrapped, its parser is implemented in load_grammar.py
+# This grammar matches that one, but does not enfore some rules that it does.
+# If you want to enforce those, you can pass the "LarkValidatorVisitor" over
+# the parse tree, like this:
+
+# import os
+# import lark
+# from lark.lark_validator_visitor import LarkValidatorVisitor
+#
+# lark_path = os.path.join(os.path.dirname(lark.__file__), 'grammars/lark.lark')
+# lark_parser = Lark.open(lark_path, parser="lalr")
```

Comment:
For exactly this kind of usecase we have `Lark.open_from_package`:

```
lark_parser = Lark.open_from_package("lark", "grammars/lark.lark")
```

This should also be used in the tests below.

Reply:
Fixed in 200d6b5f004425174d2364cf701e964c3ca2e2a0, including in the test modules.